implementations/rust/ockam/ockam_command/src/project/show.rs

```diff
@@ -40,32 +42,113 @@ impl ShowCommand {
 }
 
 async fn rpc(ctx: Context, (opts, cmd): (CommandGlobalOpts, ShowCommand)) -> miette::Result<()> {
-    run_impl(&ctx, opts, cmd).await
+    run_impl(ctx, opts, cmd).await
+}
+
+async fn run_impl(ctx: Context, opts: CommandGlobalOpts, cmd: ShowCommand) -> miette::Result<()> {
+    ProjectShowTui::run(ctx, opts, cmd.name).await
 }
 
-async fn run_impl(ctx: &Context, opts: CommandGlobalOpts, cmd: ShowCommand) -> miette::Result<()> {
-    let node = InMemoryNode::start(ctx, &opts.state).await?;
-    let controller = node.create_controller().await?;
+pub struct ProjectShowTui {
```

Comment:
Looks like you added the changes from the other PR here. Could you remove the changes done in this file?

Reply:
Yeah sorry for that 🙏, I've carried them around from the project show pr.